Buildings/Controls/OBC/UsersGuide.mo

```diff
@@ -97,6 +113,24 @@ as it is clear that this will be the room temperature.
     <td>Time constant of integrator block for cooling control loop signal</td></tr>
 <tr><td><code>TdCoo</code></td>
     <td>Time constant of derivative block for cooling control loop signal</td></tr>
+<tr><td><code>nChi</code></td>
+    <td>Total number of chillers</td></tr>
+<tr><td><code>have_parChi</code></td>
+    <td>True: the plant has parallel chillers</td></tr>
+<tr><td><code>have_heaChiWatPum</code></td>
+    <td>True: the plant has headed chilled water pumps</td></tr>
+<tr><td><code>have_fixSpeConWatPum</code></td>
+    <td>True: the plant has fixed speed condenser water pumps</td></tr>
+<tr><td><code>need_reduceChillerDemand</code></td>
+    <td>True: the plant needs to limit chiller demand when chiller staging</td></tr>
+<tr><td><code>minChiLif</code></td>
+    <td>Minimum allowable lift at minimum load for chiller</td></tr>
+<tr><td><code>TChiWatSupMin</code></td>
+    <td>Minimum chilled water supply temperature</td></tr>
+<tr><td><code>dpChiWatPumMin</code></td>
+    <td>Minimum chilled water pump differential static pressure</td></tr>
+<tr><td><code>LIFT_min</code></td>
```

Comment:
Is there a reason for having LIFT uppercase? This is not an abbreviation and hence should be lower case.